spec/libraries/acts_as_condition_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'Extension: Acts as Condition', type: :model do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+    let(:achievement) do
+      build(:achievement).tap do |a|
+        cond = build(:achievement_condition)
+        cond.conditional = a
+        cond.save
+      end
+    end
+
+    class Dummy < ActiveRecord::Base
```

Comment:
`self` here is `RSpec::ExampleGroups::ExtensionActsAsCondition::WithTenantInstance`. Wasn't sure the class should be under that namespace, plus I'd have to fully qualify it in the `subject` block below, so I used an anonymous class instead.